src/convertToObject.js

```diff
@@ -7,6 +7,29 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const regex = /\n+|\t+/g;
+
+  sourceString
+    .replaceAll(regex, ' ')
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item !== '')
+    .map((item) => {
+      return item.split(':').map((x) => {
+        return x
+          .split(',')
+          .map((y, index) => {
+            return index === 0 ? y.trim() : `          ${y.trim()}`;
+          })
+          .join(',\n');
```

Comment:
[CODE KNOWLEDGE] - There's no need to split by comma and join with a newline, as CSS properties shouldn't contain commas that need to be treated in this way. You can directly trim the property and value after splitting by the colon.